documentation/auto-discovery/topics/troubleshooting-tools.rst

```diff
@@ -0,0 +1,21 @@
+*********************
+Troubleshooting Tools
+*********************
+
+The Linux version of the data collector comes with a set of troubleshooting tools that can help in troubleshooting discovery and monitoring issues.
+
+Below is a list of those tools and their function:
+
+1. BacnetIpWalkerCli is used for BACnet/IP protocol troubleshooting
+2. IpmiScannerCli is used for IPMI protocol troubleshooting
+3. ModbusTcpWalkerCli is used for Modbus protocol troubleshooting
+4. PortScannerCli is used to ping remote protocol ports, e.g. port 22 for SSH to see if the device responds
+5. SnmpGetCli is used for SNMP protocol troubleshooting
+6. VmwareDumpCli is used for VMWare protocol troubleshooting
+
+Other general tools bundled with the data collector are:
+
+1. RegistrationCli is used to manually register the data collector and as part of the installation and reconfiguration process
```

Comment:
"RegistrationCli is used to register the Data Collector, as part of the installation and reconfiguration process, or manually."